src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -83,6 +102,25 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 		content.add(btnRunCommand, c);
 	}
 
+	private void importCommands() {
+		int returnValue = fileChooser.showOpenDialog(null);
+		if (returnValue == JFileChooser.APPROVE_OPTION) {
+			File selectedFile = fileChooser.getSelectedFile();
+			JsonParser jsonParser = new JsonParser(selectedFile);
```

Comment:
The goal is to make the design of parser reusable. Now we accept the content only from a File (what if we have a simple String? we are forced to save it to File to use parser 👎 ) and we cannot add support for other formats easily.
Moreover, we cannot reuse parser instance (we need a new one for a single parse). This all should be fixed by defining a correct interface.

Reply:
At "we cannot add support for other formats easily". There is a filter method in FileChooser and all we have to do is add format after a comma.